### PR TITLE
Distinguish between password fields

### LINKED ISSUE #54: OPUNetGameSelectWnd contains a function and class variable with the same name
We have a char[] password both as a private class variable and as a function variable in function OnClickCreate.

I was thinking maybe hostPassword and joiningPasswordAttempt or something.

### PR DESCRIPTION
**Pull request by Brett208**
Closes #54

### CODE REVIEW

File: client/OPUNetTransportLayer.cpp [hunk 1]
```diff
@@ -1146,7 +1146,7 @@ bool OPUNetTransportLayer::DoImmediateProcessing(Packet &packet, sockaddr_in &fr
 			if (tlMessage.searchQuery.gameIdentifier != gameIdentifier) {
 				return true;		// Packet handled (discard)
 			}
-			// Verify password
+			// Verify joinRequestPassword
 			if (strncmp(tlMessage.searchQuery.password, password, sizeof(password)) != 0) {
```
Comment (DanRStevens) [inline client/OPUNetTransportLayer.cpp]: Is this part correct? It seems odd the comment was updated, but the field names were not.

File: client/OPUNetTransportLayer.cpp [hunk 2]
```diff
@@ -167,7 +167,7 @@ bool OPUNetTransportLayer::HostGame(USHORT port, const char* password, const cha
 	hostedGameInfo.createGameInfo.startupFlags.b1 = 0;
 	hostedGameInfo.createGameInfo.startupFlags.missionType = gameType;
 	hostedGameInfo.createGameInfo.startupFlags.numInitialVehicles = 0;
-	// Copy the game creator name and password
+	// Copy the game creator name and joinRequestPassword
 	strncpy_s(hostedGameInfo.createGameInfo.gameCreatorName, creatorName, sizeof(hostedGameInfo.createGameInfo.gameCreatorName));
 	strncpy_s(this->password, password, sizeof(this->password));
```
Comment (DanRStevens) [inline client/OPUNetTransportLayer.cpp]: Again, the comment was updated, but none of the field names.

**Comment by Brett208** [comment]
@DanRStevens, I think this commit addresses your comments and valid concerns...

 It looks like circle ci support has been removed from the project.

**Comment by DanRStevens** [comment]
CircleCI was recently added, in preparation for the repository merge. The Server project uses CircleCI, however the Client project so far does not. (It's Windows only). The failures are because there is currently no CircleCI config file present.

I should perhaps either disable CircleCI for now, or maybe add a Mingw build.

----

I've review in detail in the morning.

**Comment by Brett208** [comment]
OK. I had assumed since there was a makefile already present that we had been compiling a circleci build in the past. I suppose I'm fine with merging things with failed circleci builds if it isn't established yet.

**Review by DanRStevens — APPROVED** [review]
Looks good.

----

I've disabled the CircleCI build for now.

**Pull request merged**